0.2.0/janusgraph.properties

```diff
@@ -0,0 +1,13 @@
+gremlin.graph=org.janusgraph.core.JanusGraphFactory
```

Comment:
Same as before. This will need to configurable in 0.3.x to support CGF. Might make sense to recommend the mounted option to easily enable users to pass multiple properties files.